src/acl/external/time_quota/ext_time_quota_acl.cc

```diff
@@ -27,17 +27,16 @@
  */
 
 #include "squid.h"
+#include "debug/Stream.h"
 #include "helper/protocol_defines.h"
+#include "sbuf/SBuf.h"
+#include "sbuf/Stream.h"
 
-#include <cstdlib>
-#include <cstring>
+static const int DEBUG_SECTION=1;
+
+#include <chrono>
```

Comment:
Remove as unused?

```suggestion
```

Reply:
yes. I had tried to use std::chrono but eventually decided it
wasn't worth the effort. Removing